pkg/sliexporter/vshnpostgresql_controller/vshnpostgresql_controller.go

```diff
@@ -99,13 +100,13 @@ func (r *VSHNPostgreSQLReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		res.Requeue = true
 		res.RequeueAfter = 30 * time.Second
 
-		if time.Now().Sub(inst.GetCreationTimestamp().Time) < r.StartupGracePeriod {
+		if time.Since(inst.GetCreationTimestamp().Time) < r.StartupGracePeriod {
 			// Instance is starting up. Postpone probing until ready.
 			return res, nil
 		}
 
 		// Create a pobe that will always fail
-		probe, err = probes.NewFailingPostgreSQL(vshnpostgresqlsServiceKey, inst.Name, inst.ObjectMeta.Labels[claimNamespaceLabel])
+		probe, err = probes.NewFailingProbe(vshnpostgresqlsServiceKey, inst.Name, inst.ObjectMeta.Labels[claimNamespaceLabel], errors.New("This probe should fail"))
```

Comment:
```suggestion
		probe, err = probes.NewFailingProbe(vshnpostgresqlsServiceKey, inst.Name, inst.ObjectMeta.Labels[claimNamespaceLabel], err)
```

If you pass the actual error in here we get more context in the logs. "This probe should fail", will not be helpful when trying to debug something.